samples/sequential-workflow-adaptive-card/csharp/README.md

```diff
@@ -0,0 +1,63 @@
+﻿# sequential-workflow-adaptive-card
+This sample illustrates sequential workflow, user specific views and upto date adaptive cards bot.
+
+## Prerequisites
+
+- Microsoft Teams is installed and you have an account
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+2) If you are using Visual Studio
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples/sequential-workflow-adaptive-card/SequentialUserSpecificFlow` folder
```

Comment:
please rename the folder name to `bot-sequential-flow-adaptive-cards`, update root readMe as well add this sample in Bot section

Reply:
changed folder name to `bot-sequential-flow-adaptive-cards` . Updated in root readme as well.